drivers/frequency/hmc7044/hmc7044.h

```diff
@@ -103,6 +104,7 @@ struct hmc7044_init_param {
 	bool		clkin1_vcoin_en;
 	bool		high_performance_mode_clock_dist_en;
 	bool		high_performance_mode_pll_vco_en;
+	bool		rf_reseeder_disable;
```

Comment:
Shoulden't be named the same? both disable or enable?

Reply:
I think it doesn't matter, only the init_param member is accessible to the user

Reply:
Functionally it's the same but I agree it should be enable, otherwise it all gets rather confusing.